executor/runner/runner.go

```diff
@@ -301,43 +301,50 @@ no_launchguard:
 		err = r.maybeSetupExternalLogger(ctx, logDir)
 		if err != nil {
 			r.logger.Error("Unable to setup logging for container: ", err)
-			r.updateStatus(ctx, titusdriver.Lost, err.Error())
+			r.updateStatusWithDetails(ctx, titusdriver.Lost, err.Error(), details)
 			return
 		}
 	} else {
 		r.logger.Info("Not starting external logger")
 	}
 
-	// TODO(fabio): Start should return Details
-	details, err := r.runtime.Details(r.container)
-	if err != nil {
-		r.logger.Error("Error fetching details for task: ", err)
-		r.updateStatus(ctx, titusdriver.Lost, err.Error())
-		return
-	} else if details == nil {
-		r.logger.Error("Unable to fetch task details")
+	if details == nil {
```

Comment:
Worth checking this before err != nil above? Otherwise, we just updated a bunch of status with nil details.

Reply:
Nah -- because details might be nil, for example, if we weren't able to start the image, because of a bad entrypoint. This here is more a sanity-check, and should never happen.

Reply:
-- In those cases, the error will get reported back to the user, and the program wont shut down entirely.